Triangulation_2/include/CGAL/Polyline_constraint_hierarchy_2.h

```diff
@@ -256,6 +256,7 @@ class Polyline_constraint_hierarchy_2
   void remove_constraint(Vertex_handle va, Vertex_handle vb)
   {
     remove_constraint(constraint_map[make_edge(va,vb)]);
+    constraint_map.erase(make_edge(va,vb));
```

Comment:
It seems there are two lookups in `constraint_map`. You can probably use a single lookup with  `map.find`.